symphony-bdk-http/symphony-bdk-http-webclient/build.gradle

```diff
@@ -0,0 +1,9 @@
+description = 'Symphony Java BDK Core Http Spring WebClient'
```

Comment:
Can we configure Jacoco for this module?

Reply:
Yes we can but I haven't imagined how we will write unittest for this module

Reply:
Using mockserver? (or wiremock which is nice too)